src/modules/dashboard/channelSelectModal.tsx

```diff
@@ -28,6 +30,11 @@ const ChannelSelectModal: FC<Props> = ({ phrasesByKey, showModal, setShowModal }
     })
     setInitialOrderStatusByMapping(selectedChannels)
     setShowModal(false)
+    const configuration = {
+      ...allState,
+      initialState: true,
+    }
+    putEtrustedConfiguration(user?.access_token as string, configuration)
```

Comment:
How is it possible that at this point "user" is null?

Reply:
Because of the Interface that already exists here in authStore it has the possibility to be null that is why it is a must to put ? because typescript shows error:

export type IAuthState = {
  user: Nullable<IAuthorizeResult>
  isAuthLoading: boolean
  isAuthFailed: boolean
  isAuthenticated: boolean
  isAuthError: boolean
  isRelogin: boolean
}